security/src/main/java/nl/nn/adapterframework/lifecycle/servlets/ActiveDirectoryAuthenticator.java

```diff
@@ -59,15 +61,19 @@ private void configure() throws FileNotFoundException {
 			throw new FileNotFoundException("unable to find LDAP role-mapping file ["+roleMappingFile+"]");
 		}
 		log.info("found rolemapping file [{}]", roleMappingURL);
-
-		setDefaultValues();
 	}
 
-	/** Set default values for legacy properties to ease application upgrades, these values will be overwritten when the appropiate setter is called. */
+	/** Set default values for legacy properties to ease application upgrades, these values can be overwritten when the appropriate setter is called. */
 	private void setDefaultValues() {
 		Environment env = getApplicationContext().getEnvironment();
-		url = env.getProperty("ldap.auth.url");
-		baseDn = env.getProperty("ldap.auth.user.base");
+		String legacyURL = env.getProperty("ldap.auth.url");
+		if(StringUtils.isEmpty(url) && StringUtils.isNotBlank(legacyURL)) {
```

Comment:
Why `isEmpty()` in one check, and `isNotBlank()` in the other?

Reply:
To prevent the legacyUrl overwriting the new url property by existing but set as an empty/whitespace value.

Reply:
Is it then OK to set an empty/whitespace value in the URL property in the configuration, to ignore legacy properties?
I was wondering about that side more than the legacy-properties.

Reply:
My intent is for users which are already using the legacy property's that it should work as-is. And if they add the new properties, it should ignore the legacy ones.
Even in the case of 'attempting' to set the new property but failing to properly do so should have preference over the default.